<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateSelect</class>
 <widget class="QDialog" name="updateSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>210</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(201, 202, 202);
}
QPushButton{
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:18px;
color:rgb(255, 255, 0);
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Control_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>0</string>
      </property>
      <property name="text">
       <string>1.主控程序升级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Motor_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>1</string>
      </property>
      <property name="text">
       <string>2.气阀程序升级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Valve_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>2</string>
      </property>
      <property name="text">
       <string>3.电机程序升级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Actuator_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>3</string>
      </property>
      <property name="text">
       <string>4.选针器升级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UI_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>5</string>
      </property>
      <property name="text">
       <string>5.人机程序升级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PYFI_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>4</string>
      </property>
      <property name="text">
       <string>6.裸氨功能板程序升级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back_pb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>返         回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
